lib/ansible/content.rb

```diff
@@ -24,5 +23,20 @@ def self.consolidate_plugin_playbooks(dir = PLUGIN_CONTENT_DIR)
         FileUtils.cp_r(Dir.glob("#{content.path}/*"), dir)
       end
     end
+
+    private
+
+    def roles_dir
+      @roles_dir ||= path.join('roles')
+    end
+
+    def requirements_file
+      @requirements_file ||= begin
+        local_file     = path.join('requirements.yml')
+        roles_dir_file = roles_dir.join('requirements.yml')
+
+        roles_dir_file.exist? ? roles_dir_file : local_file
```

Comment:
@agrare This code is technically a shim for the fact that provider plugins have the requirements.yml at a different location.  I'd like to get those all consistent, then this code can be simplified.

Reply:
Okay, looks like only nuage and lenovo have a `content/ansible_runner/requirements.yml`

Reply:
I assume it should be in `content/ansible_runner/roles/requirements.yml` ?

Reply:
Yeah, @agrare

Reply:
https://github.com/ManageIQ/manageiq-providers-nuage/pull/185
https://github.com/ManageIQ/manageiq-providers-lenovo/pull/276

Reply:
https://github.com/ManageIQ/manageiq/pull/19131